src/Spectre.libGenetic/InconsistentIndividualSizeAndTrueAmountException.h

```diff
@@ -0,0 +1,31 @@
+/*
+* InconsistentIndividualSizeAndTrueAmountException.h
+* Thrown when individual true amount is greater than size.
+*
+Copyright 2017 Grzegorz Mrukwa, Wojciech Wilgierz
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+#include "Spectre.libException/InconsistentArgumentSizesException.h"
+
+namespace Spectre::libGenetic
+{
+    class InconsistentIndividualSizeAndTrueAmountException :
+        public libException::InconsistentArgumentSizesException
```

Comment:
This exception was meant to be used when sizes of arrays, which [sizes] should be equal, are not. This should probably inhetir from a `BaseException`.